src/libs/actions/Report.ts

```diff
@@ -1235,14 +1235,12 @@ function deleteReportComment(reportID: string, reportAction: ReportAction) {
         {
             getConflictingRequests: (persistedRequests) => {
                 const conflictingCommands = (
-                    isDeletedParentAction
-                        ? [WRITE_COMMANDS.UPDATE_COMMENT]
-                        : [WRITE_COMMANDS.ADD_COMMENT, WRITE_COMMANDS.ADD_ATTACHMENT, WRITE_COMMANDS.UPDATE_COMMENT, WRITE_COMMANDS.DELETE_COMMENT]
+                    isDeletedParentAction ? [WRITE_COMMANDS.UPDATE_COMMENT] : [WRITE_COMMANDS.ADD_COMMENT, WRITE_COMMANDS.ADD_ATTACHMENT, WRITE_COMMANDS.UPDATE_COMMENT]
```

Comment:
Btw, do we not want to conditionally add UPDATE_COMMENT only when `isOptimisticAction` is true? (just like I did in my PR)

```suggestion
                    isDeletedParentAction ? [WRITE_COMMANDS.UPDATE_COMMENT] : [WRITE_COMMANDS.ADD_COMMENT, WRITE_COMMANDS.ADD_ATTACHMENT, reportAction.isOptimisticAction ? WRITE_COMMANDS.UPDATE_COMMENT : '']
```

What will happen if we always include it?
1. Add a comment while online (added to server)
2. Go offline
3. Edit the comment. At this point, the persisted request contains UPDATE_COMMENT
4. Delete the comment.
Both edit and delete request is cancelled out, the message is optimistically deleted, but the message will reappears when reopening the chat.